config/initializers/convert_time_to_eo.rb

```diff
@@ -0,0 +1,16 @@
+# temporary hack for when/if we can get patch accepted upscream
+# et-orbi doesn't support comparisons of EoTime in the scheduler with user provided Time objects
+# see https://github.com/ManageIQ/manageiq/pull/19153#issuecomment-521719495
+module ConvertTimeToEoTime
+  def ==(*args)
+    o = if args.kind_of?(Time)
+      EtOrbi.make_time(args.first)
+    else
+      EtOrbi::EoTime.new(args.first, "UTC")
```

Comment:
What's the reasoning for this else?

Reply:
without it, line 7 of this doesn't work when https://github.com/ManageIQ/manageiq/blob/master/spec/models/miq_schedule_worker/runner_spec.rb#L140 runs because we don't have a time object